dockers/docker-snmp/start.sh

```diff
@@ -2,6 +2,8 @@
 
 mkdir -p /etc/ssw /etc/snmp
 
+python /usr/bin/snmp_yml_to_configdb.py
```

Comment:
Along with the comment above, since your script has a shebang, as long as it is executable, the interpreter will use the program specified in the shebang to run the file, so you don't need to add `python` here. Also suggest adding a brief comment describing what this script does, like:

```bash
# Parse snmp.yml and insert the data in Config DB
/usr/bin/snmp_yml_to_configdb.py
```

Reply:
Added.

Reply:
I see this comment is still applicable.

---
In reply to: [542961334](https://github.com/Azure/sonic-buildimage/pull/6205#discussion_r542961334) [](ancestors = 542961334)